frontend/src/components/inputs/FileInput.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+function FileInput({ label, onChange, value }) {
+  const imagestyle = {
+    height: "90px",
+    width: "90px",
+  };
+  return (
+    <div>
+      <label>
+        {label}
+        <input
+          type="file"
+          onChange={(e) => {
+            onChange(label, e.target.files[0]);
+          }}
+        />
+        <img
+          src={typeof value === "object" ? URL.createObjectURL(value) : value}
```

Comment:
This currently supports only image upload? maybe we can do it generic, and add `isImage` prop, then render this `<img>` based on that

Reply:
Would we need files that aren't images?

Reply:
I doubt it

Reply:
But it's a simple change and it would make it more generic so why not